pkg/gopass/secrets/secparse/parse.go

```diff
@@ -25,14 +26,14 @@ func Parse(in []byte) (gopass.Secret, error) {
 		debug.Log("parsed as YAML: %+v", s)
 		return s, nil
 	}
-	debug.Log("failed to parse as YAML: %s\n%s", err, string(in))
+	debug.Log("failed to parse as YAML: %s\n%s", err, out.Secret(string(in)))
 	s, err = secrets.ParseKV(in)
 	if err == nil {
 		debug.Log("parsed as KV: %+v", s)
```

Comment:
That seems like a lot of logging too, no?

Reply:
Great catch! The MIME parser would indeed still leak secrets.

Error logging is quite verbose, yes. But it can be controlled with `GOPASS_DEBUG_FUNCS` and `GOPASS_DEBUG_FILES`.

Reply:
I'm not sure how this is resolved? 

This is when `err == nil`, so we have the whole secret `s` that's logged here upon success.

Reply:
But it's wrapped in an `out.Secret`. In https://github.com/gopasspw/gopass/pull/1886/files#diff-2769ebba150a804d5515d28f28b7a8e06f05d914e9781b4e58beac1dc4d2ed52R203 the string is rewritten to not contain ... anything.

Reply:
Oh right. That's a nice way to do it.